src/abstracts/SablierMerkleBase.sol

```diff
@@ -69,6 +72,13 @@ abstract contract SablierMerkleBase is
         ipfsCID = params.ipfsCID;
         MERKLE_ROOT = params.merkleRoot;
         NAME = bytes32(abi.encodePacked(params.name));
+
+        // If the shape string exceeds 32 bytes, truncate it to prevent `claim` from reverting.
+        if (bytes(params.shape).length > 32) {
+            shape = string(abi.encodePacked(bytes32(abi.encodePacked(params.shape))));
+        } else {
+            shape = params.shape;
```

Comment:
what if we use the same approach for  campaign name? store it as string and truncate it have only 32 bytes

we can add a helper function `truncateSToB32`

Reply:
Sounds good to me.